myhpom/templates/myhpom/dashboard.html

```diff
@@ -1,11 +1,114 @@
 {% extends "myhpom/base.html" %}
 
 {% block main_content %}
-<section class="jumbotron text-center">
-    <div class="container">
-        <h1 class="jumbotron-heading">Dashboard</h1>
-        <p>Real Dashboard Coming Soon!</p>
+<div class="row">
+    <div class="col-sm-12">
+        <h1 class="h6 dashboard__title">
+            My dashboard
+        </h1>
+
+        <section class="dashboard-user-info">
+            <div class="h1 dashboard-user-info__text">
+                {{ request.user.get_full_name }}
+            </div>
+            <div class="h6 dashboard-user-info__text">
+                {{ request.user.email }}
+            </div>
+        </section>
     </div>
-</section>
-{% endblock main_content %}
+</div>
+<div class="row">
+    <div class="col-md-8 dashboard-main">
+        <hr/>
+
+        <section class="dashboard__section dashboard-profile-information">
+            <div class="float-right">
+                <a href="/" class="small">Edit My Profile</a>
```

Comment:
These float-right divs appear to be pushing the anchors down a little bit compared to its title corresponding title.